### ISSUE TITLE
Homebrew game working on the online version but not the downloadable one.

### ISSUE THREAD

**Issue by zdizzler** [endrift/gbajs#43]
Hey, I checked out your emulator and was excited to see that my game actually worked on it! However, when I downloaded it and ran it through index.html, it crashes immediately. I narrowed the problem down to the CPU's first step, which is strange as I figured this wouldn't be different between the two versions, so I think I'm missing something. The game is on GitHub here: https://github.com/zdizzler/TheCave and you should only need the .gba file, just in case you wanted to test it out yourself. Thanks!

**Comment by endrift** [comment]
Trying to run GBA.js from the file: scheme can cause issues with browsers due to their security model. It's probably failing to load the BIOS, which is not a GBA.js bug. Try seeing if console.html reports any errors.

**Comment by zdizzler** [comment]
Yeah there's some stuff there (the reason I'm doing this is to make the game playable on my personal website by embedding the emulator):
![capture](https://cloud.githubusercontent.com/assets/5561576/9151049/dd684984-3dc2-11e5-8c01-83c290721d94.PNG)

**Comment by endrift** [comment]
I said console.html, not the web inspector console.

**Comment by zdizzler** [comment]
Nothing goes wrong, other than the screen being a bit distorted/overscaled, but I'm guessing this is just an artifact of your program? Otherwise, no errors were reported.
